twill-core/src/main/java/org/apache/twill/internal/AbstractTwillController.java

```diff
@@ -170,6 +171,37 @@ public String apply(Set<String> input) {
     });
   }
 
+  @Override
+  public Future<LogEntry.Level> setRootLogLevel(LogEntry.Level logLevel) {
+    return sendMessage(SystemMessages.setLogLevels(ImmutableMap.of(Logger.ROOT_LOGGER_NAME, logLevel)), logLevel);
+  }
+
+  @Override
+  public Future<LogEntry.Level> setRootLogLevel(String runnableName, LogEntry.Level logLevel) {
+    return sendMessage(SystemMessages.setLogLevels(runnableName, ImmutableMap.of(Logger.ROOT_LOGGER_NAME, logLevel)),
+                       logLevel);
+  }
+
+  @Override
+  public Future<Map<String, LogEntry.Level>> setLogLevels(Map<String, LogEntry.Level> logLevels) {
+    return sendMessage(SystemMessages.setLogLevels(logLevels), logLevels);
+  }
+
+  @Override
+  public Future<Map<String, LogEntry.Level>> setLogLevels(String runnableName,
+                                                          Map<String, LogEntry.Level> runnableLogLevels) {
+    return sendMessage(SystemMessages.setLogLevels(runnableName, runnableLogLevels), runnableLogLevels);
+  }
+
+  @Override
+  public Future<String> resetLogLevelsToDefault() {
+    return sendMessage(SystemMessages.resetLogLevels(null), "all");
```

Comment:
This is inconsistent with the `setLogLevels` method, which doesn't require explicitly passing in `null` for all runnables case.